src/ray/pubsub/publisher.cc

```diff
@@ -233,17 +253,20 @@ std::unique_ptr<EntityState> SubscriptionIndex::CreateEntityState() {
   switch (channel_type_) {
   case rpc::ChannelType::RAY_ERROR_INFO_CHANNEL:
   case rpc::ChannelType::RAY_LOG_CHANNEL: {
-    return std::make_unique<CappedEntityState>();
+    return std::make_unique<EntityState>(
```

Comment:
Do you think we should map out all the values of the enum in this PR or separate?

Reply:
Ah I thought you were going to merge that as a separate PR. But I can add it here as well if that's easier.